docs/proposals/job_controller.md

```diff
@@ -0,0 +1,191 @@
+# Job Controller
+
+## Abstract
+A proposal for implementing a new controller - Job controller, which will be responsible
+for managing pods that require to run:
+* once on demand or at a specified point in time
+* repeatedly at a specified point in time.
+
+Several exiting issues were already created regarding that particular subject:
+* Distributed CRON jobs [#2156](https://github.com/GoogleCloudPlatform/kubernetes/issues/2156)
+* Job Controller [#1624](https://github.com/GoogleCloudPlatform/kubernetes/issues/1624)
+
+
+## Use Cases
+1. Ability to have a controller that will create a run-once pod.
+1. Ability to have a controller that will have an option to specify reaction on pod’s success/failure.
+1. Ability to specify number of restarts upon pod failure.
+1. Ability to specify maximum running time for a pod.
+1. Ability to create a repetitive job, eg. database backup, sending emails.
+1. Ability to create one time job to be run at a specified time, eg. system analysis.
+1. Ability to create a chain of jobs dependent one on another.
```

Comment:
Frankly said I didn't even think about any of those use cases, somehow I missed it. I'll try to catch you on IRC to talk it through.